org.github.Powder-Toy.yaml

```diff
@@ -0,0 +1,91 @@
+app-id: org.github.Powder-Toy
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: powder-toy
+rename-icon: powder
+rename-desktop-file: powder.desktop
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--device=dri"
+  - "--socket=wayland"
+  - "--share=network"
+  - "--persist=."

```

Comment:
why it does need a . persist here?

Reply:
Because file powder.pref is created on first run. Using persist=powder.pref don't work, it's create directory named powder.pref.